molgenis-dataexplorer/src/main/resources/js/dataexplorer.js

```diff
@@ -380,6 +380,12 @@ function($, molgenis, settingsXhr) {
 			$(document).trigger('removeAttributeFilter', {'attributeUri': $(this).data('href')});
 		});
 		
+		$('#delete').on('click', function(e){
+			$.ajax('/api/v1/'+selectedEntityMetaData.name+'/meta', {'type': 'DELETE'}).done(function(){
```

Comment:
How about adding a confirm dialog that says "Are you sure you want to delete all data and meta data for entity <entity>"?